cristin-commons/src/main/java/no/unit/nva/cristin/common/model/cristin/CristinInstitution.java

```diff
@@ -37,5 +42,19 @@ public String getUrl() {
     public void setUrl(String url) {
         this.url = url;
     }
+
+    /**
+     * Build a Organization datamodel from a CristinInstitution datamodel.
+     *
+     * @return a Organization converted from a CristinInstitution
+     */
+    public Organization toOrganization() {
+
+        return new Organization.Builder()
+                .withId(buildUri(Constants.CRISTIN_API_BASE_URL, Constants.INSTITUTION_PATH,
```

Comment:
Maybe refactor some day using UriWrapper, but not very important

Reply:
Are you using more than one UriUtils?

Reply:
Merged to one